### PR TITLE
49 use starter ts as the starting point

### PR DESCRIPTION
**Pull request by ComLock**

### CODE REVIEW

File: docs/project.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+It comes with everything you need to start writing tests with Jest.
+
+TIP: If you want to learn how to add Jest to an existing project, read the https://developer.enonic.com/docs/starter-ts/stable/test[Testing] chapter of the https://developer.enonic.com/docs/starter-ts/stable[TypeScript Starter] documentation.
+
+To create a project for the tutorial, run the following command:
+
+[source, Terminal]
+----
+enonic project create jest-tutorial -r starter-ts
```
Comment (alansemenov) [inline docs/project.adoc:15]:
```suggestion
enonic create jest-tutorial -r starter-ts
```

File: docs/setup.adoc [hunk 2]
```diff
@@ -9,7 +9,10 @@ You may follow the steps in this guide to configure Jest for an existing project
 
 To create a project for the tutorial, run the following command:
 
-  enonic project create com.example.tutorial.jest -r tutorial-jest -d jest-tutorial
+[source, Terminal]
+----
+enonic project create jest-tutorial -r starter-ts
```
Comment (alansemenov) [inline docs/setup.adoc:14]:
```suggestion
enonic create jest-tutorial -r starter-ts
```

File: tsup.config.ts [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-		return import('./tsup/client-static').then(m => m.default());
-	}
-	throw new Error(`Unconfigured directory:${options.d}!`)
-})
+export default defineConfig(async (options: Options) => {
+  if (options.d === DIR_DST) {
+    return import('./tsup/server').then(m => m.default());
+  }
+  if (options.d === DIR_DST_ASSETS) {
+    return import('./tsup/client-asset').then(m => m.default());
```
Comment (alansemenov) [inline tsup.config.ts:13]:
```suggestion
    return import('./tsup/client').then(m => m.default());
```

Comment (alansemenov) [reply tsup.config.ts:13]: Since we're not going to have static for now it would be less confusing to call it simply "`client`".

**Review by alansemenov — APPROVED** [review]

File: tsup.config.ts (hunk 3 above)
Comment (ComLock) [reply tsup.config.ts:13]: That is fine, but that comes from starter-ts

**Pull request merged**